### PR TITLE
add shanbay trigger

### PR DESCRIPTION
**Pull request by mengziin**
It's a shanbay trigger implementation. The train of thought is the get_up.py . 

When I finished reciting words today, I trigger the get_shanbay.yml to execute the github action, add an issue comment in the "Shanbay" label issue.

please review the codes.

image

### CODE REVIEW

File: get_shanbay.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import argparse
+import requests
+import pendulum
+
+from github import Github
+
+# the shanbay issue index is 12
+GET_UP_ISSUE_NUMBER = 12
+# update your own shanbay user_name
+SHANBAY_USERNAME = "zlurjj"
```
Comment (yihong0618) [inline get_shanbay.py:10]: empty string here, i will change it to mine

**Review by yihong0618 — APPROVED** [review]
thanks

File: get_shanbay.py (hunk 1 above)
Comment (mengziin) [reply get_shanbay.py]: already setting to empty , please check out it ~

**Comment by yihong0618** [comment]
Thanks, welcome to open source world.

**Pull request merged**